addons/mail/static/src/core/common/message.js

```diff
@@ -134,16 +134,16 @@ export class Message extends Component {
         );
         useEffect(
             () => {
-                if (!this.shadowRoot) {
+                if (!this.shadowRoot || !this.message.type || this.message.type.includes("email")) {
                     return;
                 }
-                if (this.state.originalFormat) {
-                    this.shadowRoot.removeChild(this.shadowStyle);
-                } else {
+                if (cookie.get("color_scheme") === "dark") {
```

Comment:
Is white and dark possibly dynamically swapped?
Otherwise it feels like "white" could just be an extra condition for the quick return at the top, not needing to remove the child.

Reply:
That's what I thought at first, but no: inserting `shadowStyle` means the style has added alteration from the shadowDOM encapsulation. So this is correct to alter in "dark" and not in white

Reply:
Yes but why is white removing the style?
While all the other conditions at the start of the method do nothing?
The style is always added by default? And it needs to be removed for white email?

Reply:
Removing it, as it made sense with toggler button that has been removed